build.gradle

```diff
@@ -331,6 +331,7 @@ subprojects { subproject ->
 
 	checkstyle {
 		configFile = file("$rootDir/src/checkstyle/checkstyle.xml")
+		configDir = file("$rootDir/src/checkstyle/")
```

Comment:
Hm. Doesn't this mean that you fully bypass Checkstyle?
Your import order is wrong and it definitely has to fail.

Please, consider to revert Checkstyle changes and fix your code according requirements of our checks.
Thanks

Reply:
I removed this code, modified locally the xml file.